deploy-dotorg.sh

```diff
@@ -36,6 +36,9 @@ rm -rf ./deploy
 # Do things for all of the themes
 for THEME_SLUG in */ ; do
 
+	# Remove slashes from the theme slug
+	THEME_SLUG=${THEME_SLUG%/}
```

Comment:
Going to one-up my own suggestion, and suggest this other snippet, which removes _all_ slashes

```suggestion
	THEME_SLUG=$(echo $slug | tr -d '/')
```

Reply:
Nice! Is `$slug` a new variable? Or should this be `THEME_SLUG=$(echo $THEME_SLUG | tr -d '/')`?

Reply:
Looks like you updated the suggestion, I'll test it out, thanks!